api/chunks/views.py

```diff
@@ -0,0 +1,45 @@
+from http import HTTPStatus
+from typing import List
+
+from django.http import HttpRequest
+from ninja import Router
+from ninja.pagination import LimitOffsetPagination, paginate
+
+from chunks.models import Chunk
+from chunks.schemas import ChunkIn, ChunkOut
+import json
+
+chunk_router = Router()
+
+@chunk_router.post("/chunk/", response={HTTPStatus.CREATED: ChunkOut})
+def create_chunk(request: HttpRequest, payload: ChunkIn):
+    chunk = Chunk(**payload.dict())
+    chunk.full_clean()
+    chunk.save()
+    return HTTPStatus.CREATED, chunk
```

Comment:
**issue:** separate logic (controllers) from the views - create separate file with functions that map request's input to response output and just run them in here